cmd/surf/commands/createdocument.go

```diff
@@ -0,0 +1,112 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"github.com/CloudHub360/ch360.go/ch360"
+	"github.com/CloudHub360/ch360.go/config"
+	"github.com/pkg/errors"
+	"gopkg.in/alecthomas/kingpin.v2"
+	"os"
+	"strconv"
+)
+
+type CreateDocumentCmd struct {
```

Comment:
❓ Just pondering what use cases the inclusion of this command supports, given that document operations automatically create documents under the covers. I see it's good for completeness, but is it useful beyond that?

Reply:
Currently, it's useful only for experimenting and getting the metadata (filetype etc) about a file. I do see it being useful to be able to specify a doc ID as opposed to a filename when doing the `read`/`classify`/`extract` commands, though, at some point in the not-too-distant.

Eg 
```
surf create document blah.pdf
# (get doc-id)
surf extract names the-doc-id
surf classify business-docs-classifier the-doc-id
surf delete document the-doc-id
```